src/prevector.h

```diff
@@ -394,6 +394,21 @@ class prevector {
         fill(ptr, first, last);
     }
 
+    inline void resize_uninitialized(size_type new_size) {
+        // resize_uninitialized change the size of the prevector but does not initialize.
```

Comment:
`// resize_uninitialized changes the size of the prevector but does not initialize it`